### ISSUE TITLE
[Question] Is this library still being maintained?

### ISSUE THREAD

**Issue by KingAmo** [facebookexperimental/Recoil#2324]
any news here?

**Comment by Dxuian** [comment]
been a week since you asked so ig no

**Comment by xnovak1** [comment]
Its weird that last merge to main was 2 years ago..

**Comment by bowheart** [comment]
This has been asked several times. The most prominent asking is [here](https://github.com/facebookexperimental/Recoil/issues/2288).

Recoil has been dead for years. Jotai is a lightweight alternative. Zedux is a more direct replacement. Both these libs have expanded on Recoil's ideas and are much better at this point.

**Comment by maggiben** [comment]
Oh boy here we go...

**Comment by AnirudhMemani** [comment]
Looks like it's dead. Most of my searches return page not found in the docs.

**Comment by centauricoder01** [comment]
I guess, even though it's dead. A lot of people are using it.

**Comment by LOGANLEEE** [comment]
> I guess, even though it's dead. A lot of people are using it.

yeah I agree, recoil is the best lib so far for its purpose but I guess have to let it go.

**Comment by seungwubaek** [comment]
Agree. bye recoil... RIP. He was.. one of the my favs and the best lib to manage states.
I write the last code of recoil on my project now, and it's really time to let him go.

bye. I'll miss you.

Please tell me any stable alternatives 🙏 thanks

**Comment by deanilvincent** [comment]
@seungwubaek You can try redux toolkit :)
